applications/matter_bridge/sample.yaml

```diff
@@ -26,3 +26,9 @@ tests:
     integration_platforms:
       - nrf7002dk_nrf5340_cpuapp
     platform_allow: nrf7002dk_nrf5340_cpuapp
+  applications.matter_bridge.smp_dfu.br_ble:
```

Comment:
can you check in your code using ctf_tm -c spec -r  build, if this is correct name of the sample?

Reply:
I checked, it works fine